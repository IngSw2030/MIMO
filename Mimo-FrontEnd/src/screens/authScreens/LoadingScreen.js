import React from 'react';
import { useContext } from 'react';
import { View, Text, StyleSheet, Image } from 'react-native';
import { Context as PurchaseContext } from '../../context/PurchaseContext';
import { Context as ServiceContext } from '../../context/ServiceContext';
import { Context as UserContext } from '../../context/UserContext';
import { Context as VetContext } from '../../context/VetContext';
import { Context as SellsContext } from '../../context/SellsContext';
import { Context as ProductContext } from '../../context/ProductContext';
import { Context as ShoppingCartContext } from '../../context/ShoppingCartContext';
import { useEffect } from 'react';
import { navigate } from '../../navigationRef';
import { withNavigation } from 'react-navigation';




const LoadingScreen = ({ navigation }) => {
	// const { getPosts } = useContext(PostContext);
	const { getMyPurchases } = useContext(PurchaseContext);
	const { getMySells } = useContext(SellsContext);
	const { state, getUser } = useContext(UserContext);
	const { getMyShopingCart } = useContext(ShoppingCartContext);
	const { getAllVets } = useContext(VetContext);
	const { getProduct } = useContext(ProductContext);

	const mimo = require('../../../assets/mimo.png')

	// const { getPet } = useContext(PetContext);

	useEffect(() => {
		const fetchData = async () => {
			if (state.tipo === null) {
				getUser();
				getAllVets({ name: "" });
				getProduct({ name: "", pets: '' });
				getMyPurchases(); //Posible mejora: hacer solo una llamada de purchase y filtrar la lista aqui.
				getMySells();
				getMyShopingCart();
			} else {
				state.tipo == false ?
					navigation.navigate('HomePage')
					: navigation.navigate('comercHome')
			}

		}
		fetchData();
	}, [state]); // Or [] if effect doesn't need props or state

	return (
		<View style={styles.general}>
			<View style={{
				flex: 1, alignItems: 'center',
				justifyContent: 'center'
			}}>
				<Image style={styles.image} source={mimo} />
			</View>
			<Text style={styles.texto}>Zorah Inc 2020</Text>
		</View>
	);
};

const styles = StyleSheet.create({
	general: {
		flex: 1,
		flexDirection: 'column',
		backgroundColor: '#FFF7BB',

	},
	image: {
		height: 250,
		width: 250,
	},
	texto: {
		textAlign: 'center',
		fontSize: 20
	}
});

export default withNavigation(LoadingScreen);
