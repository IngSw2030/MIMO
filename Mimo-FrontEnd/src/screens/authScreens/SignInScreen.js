import React, { useContext } from 'react';
import { View, StyleSheet, Text, Image } from 'react-native';
import { NavigationEvents } from 'react-navigation';
import AuthForm from '../../components/authForm';
import NavLink from '../../components/navLink';
import { Context } from '../../context/AuthContext';

const SigninScreen = () => {
  const { state, signin, clearErrorMessage } = useContext(Context);
  const mimoIcon = require('../../../assets/mimo.png');

  return (
    
    <View style={styles.screenContainer}>
    <Image style={styles.logoStyle} source={mimoIcon} />    
      <View style={styles.authContainer}>
        <NavigationEvents onWillBlur={clearErrorMessage} />
        <AuthForm
          headerText="BIENVENID@"
          errorMessage={state.errorMessage}
          onSubmit={signin}
          submitButtonText="¡Hecho!"
          titulo="titulo"
        />
        <NavLink
          text="¿Aún no tiene una cuenta? "
          navText="¡Regístrese acá!"
          routeName="Signup"
        />
      </View>
    </View>
  );
};

SigninScreen.navigationOptions = () => {
  return {
    headerShown: false
  };
};

const styles = StyleSheet.create({
    logoStyle: {
        marginTop: 30,
        height: 250,
        width: 200,
        alignSelf: 'center',
    },
    screenContainer:{
    flex: 1,
    backgroundColor: '#FFF7DB'
    },  
    authContainer: {
    justifyContent: 'center',
    marginBottom: 250,
    marginTop: 1,
    }
});

export default SigninScreen;