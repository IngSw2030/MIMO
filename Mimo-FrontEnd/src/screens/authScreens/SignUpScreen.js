import React, { useContext } from 'react';
import { View, StyleSheet, Image } from 'react-native';
import { NavigationEvents } from 'react-navigation';
import AuthForm from '../../components/authForm';
import NavLink from '../../components/navLink';
import { Context } from '../../context/AuthContext';

const SignupScreen = ({ navigation }) => {
  const { state, signup, clearErrorMessage } = useContext(Context);
  const mimoIcon = require('../../../assets/mimo.png');

  return (
    <View style={styles.screenContainer}>
    <Image style={styles.logoStyle} source={mimoIcon} /> 
      <View style={styles.authContainer}>
        <NavigationEvents onWillBlur={clearErrorMessage} />
        <AuthForm
          headerText="¡REGÍSTRATE!"
          errorMessage={state.errorMessage}
          submitButtonText="¡Hecho!"
          onSubmit={signup}
        />
        <NavLink
          routeName="Signin"
          text="¿Ya tiene una cuenta? "
          navText="¡Ingrese acá!"
        />
      </View>
    </View>
  );
};

SignupScreen.navigationOptions = () => {
  return {
    headerShown: false
  };
};

const styles = StyleSheet.create({
    logoStyle: {
        marginTop: 30,
        height: 250,
        width: 200,
        alignSelf: 'center',
    },
    screenContainer:{
    flex: 1,
    backgroundColor: '#FFF7DB'
    },  
    authContainer: {
    justifyContent: 'center',
    marginBottom: 250,
    marginTop: 1,
    }
});

export default SignupScreen;