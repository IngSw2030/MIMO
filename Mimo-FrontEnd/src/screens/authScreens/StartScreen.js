import React, { useContext, useEffect } from 'react';
import { View, Text, StyleSheet, Image } from 'react-native';
import { TouchableOpacity } from 'react-native-gesture-handler';
import { Context as AuthContext } from '../../context/AuthContext';
import { Context as UserContext } from '../../context/UserContext';

const StartScreen = ({ navigation }) => {
	const mimoIcon = require('../../../assets/mimo.png');

	const { getUser } = useContext(UserContext);
	const { tryLocalSignin } = useContext(AuthContext);
	useEffect(() => {
		tryLocalSignin();
		getUser();
	}, []);

	return (
		<View>
			<Image style={styles.logoStyle} source={mimoIcon} />
			<View style={styles.buttonContainer}>
				<TouchableOpacity style={styles.roundedContainerStyle1} onPress={() => navigation.navigate('Signin')}>
					<Text style={styles.buttonTextStyle}>INICIAR SESIÃ“N ''</Text>
				</TouchableOpacity>
				<TouchableOpacity style={styles.roundedContainerStyle2} onPress={() => navigation.navigate('Signup')}>
					<Text style={styles.buttonTextStyle}>REGISTRARSE</Text>
				</TouchableOpacity>
				<TouchableOpacity onPress={() => navigation.navigate('Signup')}>
					<Text style={styles.registerEntStyle}>Registrarse como empresa</Text>
				</TouchableOpacity>
				<TouchableOpacity onPress={() => navigation.navigate('Join')}>
					<Text style={styles.registerEntStyle}>Ir al chat</Text>
				</TouchableOpacity>
			</View>
		</View>
	);
};

StartScreen.navigationOptions = () => {
	return {
		headerShown: false,
	};
};

const styles = StyleSheet.create({
	screenContainer: {
		flex: 1,
		backgroundColor: '#FFF7BB',
	},
	buttonContainer: {
		justifyContent: 'center',
		marginBottom: 200,
		marginTop: 1,
	},
	logoStyle: {
		marginTop: 60,
		height: 250,
		width: 250,
		alignSelf: 'center',
	},
	roundedContainerStyle1: {
		marginTop: 20,
		alignSelf: 'center',
		backgroundColor: '#88CCF2',
		height: 60,
		width: 280,
		borderRadius: 75,
		justifyContent: 'center',
	},
	roundedContainerStyle2: {
		marginTop: 30,
		alignSelf: 'center',
		backgroundColor: '#E8778B',
		height: 60,
		width: 280,
		borderRadius: 75,
		justifyContent: 'center',
	},
	buttonTextStyle: {
		fontSize: 30,
		alignSelf: 'center',
		fontWeight: '300',
	},
	registerEntStyle: {
		fontSize: 15,
		marginTop: 25,
		color: '#005CCA',
		alignSelf: 'center',
	},
});
export default StartScreen;
