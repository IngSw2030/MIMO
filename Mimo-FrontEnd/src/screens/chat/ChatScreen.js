import React from 'react';
import { useState } from 'react';
import { useEffect } from 'react';
import { View } from 'react-native';
import { GiftedChat } from 'react-native-gifted-chat';
import { useDispatch, useSelector } from 'react-redux';

ChatScreen.navigationOptions = screenProps => ({
	title: screenProps.navigation.getParam('name'),
});

export default function ChatScreen({ navigation }) {
	const dispatch = useDispatch();
	const selfUser = useSelector(state => state.selfUser);
	const chatState = useSelector(state => state);

	const userId = navigation.getParam('userId');

	const messages = useSelector(state => state.conversations).find(
		thisConversation => thisConversation.conversationId === userId
	).messages;
	console.log('conversations en ChatScreen', messages);
	return (
		<View style={{ flex: 1 }}>
			<GiftedChat
				messages={messages}
				onSend={messages => {
					dispatch({
						/* Agrega localmente el mensaje */
						type: 'private_message',
						/* userId es el email del receptor */
						data: { message: messages[0], conversationId: userId },
					});
					dispatch({
						/* Manda el mensaje al servidor para enviarlo al receptor */
						type: 'server/private_message',
						/* userId es el email del receptor */
						data: { message: messages[0], conversationId: userId },
					});
				}}
				user={{
					//indica si es mensaje propio o de un emisor
					_id: selfUser.userId,
				}}
			/>
		</View>
	);
}
