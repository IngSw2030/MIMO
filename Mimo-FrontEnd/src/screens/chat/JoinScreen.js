import React, { useState } from 'react';
import { useContext } from 'react';
import { View, TextInput, Image, Button } from 'react-native';
import { useDispatch } from 'react-redux';
import { Context as UserContext } from '../../context/UserContext';

export default function JoinScreen({ navigation }) {
	const dispatch = useDispatch();
	const [username, setUsername] = useState('');
	const { state: user } = useContext(UserContext);
	return (
		<View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
			<Image resizeMode='contain' style={{ flex: 1 }} source={require('../../../assets/mimo.png')} />
			<View style={{ flex: 1, justifyContent: 'space-around' }}>
				<TextInput
					onChangeText={text => setUsername(text)}
					value={username}
					style={{ fontSize: 30, textAlign: 'center' }}
					placeholder='Enter username'
				/>
				<Button
					title='Join Chat'
					onPress={() => {
						dispatch({ type: 'server/setUser', data: user.email });
						dispatch({ type: 'server/join', data: user.name });
						navigation.navigate('FriendList');
					}}
				/>
			</View>
		</View>
	);
}
