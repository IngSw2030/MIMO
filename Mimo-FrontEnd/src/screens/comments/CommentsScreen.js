import React, {useState, useContext, useEffect} from 'react';
import { View, Text, StyleSheet, TextInput,FlatList } from 'react-native';
import { TouchableOpacity } from 'react-native-gesture-handler';
import { navigate } from '../../navigationRef';
import { Context as CommentContext } from '../../context/CommentContext';
import { FontAwesome5 } from '@expo/vector-icons';
import CommentComponent from '../../components/commentComponent';

const commentsScreen = ({ navigation }) => {

	const id = navigation.getParam('id');
	const [comment, setComment] = useState('');
	const {state, saveComment} = useContext(CommentContext);

	const [flag, setFlag] = useState(false);

	const handleSend = () => {
		saveComment({content: comment, idPost: id});
		setComment('');
		setFlag(true);
	}

	useEffect(() => {
	}, [handleSend])

	return (
		<View style={styles.general}>
			<Text style={styles.titles}> Comentarios </Text>
			<View style={styles.topPart}>
				<View style = {styles.inputComment}>
					<TextInput 
						placeholder="Escribe tu comentario"
						placeholderTextColor="#000"
						style={styles.inputCommentText}
						value={comment}
						editable={true}
      					multiline={true}
						onChangeText={(newComment) => setComment(newComment)}
					/>
				</View>
				<TouchableOpacity onPress={()=>{
					handleSend();
				}}>
					<View style={styles.send}>
						<FontAwesome5 name="paper-plane" size={25} color="black" />
					</View>
				</TouchableOpacity>
			</View>
			<FlatList
				showsVerticalScrollIndicator={false}
				data={state.comments}
				keyExtractor={(result) => result._id}
				renderItem={({ item }) => {
					return <CommentComponent post = {item}/>
				}}
			/>
		</View>
	);
};


styles = StyleSheet.create({
	general: {
		flex: 1,
		flexDirection: 'column',
		backgroundColor: '#FFF7BB',
	},
	titles: {
		fontSize:34,
		fontWeight: 'bold',
		padding: 15,
	},
	inputComment:{
        backgroundColor: "#7E9FD1",
        minHeight: 50,
        width: 310,
        borderRadius: 25,
		alignSelf: 'center',
		flexDirection: 'row',
		alignContent: 'center',
		margin: 10
	},
	inputCommentText:{
		justifyContent: 'flex-start',
		padding: 8,
		fontSize: 18,
		fontWeight:'600',
		flexWrap: 'wrap',
		minHeight: 50,
	},
	topPart:{
		flexDirection: 'row',
	},
	send:{
		backgroundColor: "#BCDB89",
        height: 50,
        width: 50,
        borderRadius: 25,
		alignSelf: 'center',
		alignContent: 'center',
		justifyContent: 'center',
		alignItems: 'center',
		marginVertical: 10,
		marginRight: 10
	}
});

export default (commentsScreen);
