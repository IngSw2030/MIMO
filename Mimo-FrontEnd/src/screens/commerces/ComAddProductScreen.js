import React from 'react'
import { View, Text } from 'react-native'

const ComAddProductScreen = () => {
    return (
        <View>
            <Text>Pantalla de ComAddProductScreen</Text>
        </View>
    )
}

export default ComAddProductScreen