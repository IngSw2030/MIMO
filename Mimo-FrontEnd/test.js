import HomePageScreen from './src/screens/users/HomeScreen';
import ComHomeScreen from './src/screens/commerces/ComHomeScreen';

// authScreens
import SignInScreen from './src/screens/authScreens/SignInScreen';
import SignUpScreen from './src/screens/authScreens/SignUpScreen';
import StartScreen from './src/screens/authScreens/StartScreen';

//reviews
import ReviewScreen from './src/screens/users/reviews/ReviewsScreen';

// Chat
import ChatScreen from './src/screens/chat/ChatScreen';
import JoinScreen from './src/screens/chat/JoinScreen';
import FriendListScreen from './src/screens/chat/FriendsListScreen';

//Blog/Post
import PostScreen from './src/screens/users/blog/PostScreen';
import PostDetailScreen from './src/screens/users/blog/PostDetailScreen';
import AddPostScreen from './src/screens/users/blog/AddPostScreen';
import myPostsScreen from './src/screens/users/blog/myPostsScreen';
import pinnedPostsScreen from './src/screens/users/blog/pinnedPostsScreen';
import recommendedPostsScreen from './src/screens/users/blog/recommendedPostScreen';
//Comments
import CommentsScreen from './src/screens/users/blog/CommentsScreen';

//Imports de comercio
import ComAddProductScreen from './src/screens/commerces/ComAddProductScreen';
import ComAddServiceScreen from './src/screens/commerces/ComAddServiceScreen';
import ComVeterinaryScreen from './src/screens/commerces/ComeVeterinaryScreen';
import ComAddVeterinaryScreen from './src/screens/commerces/ComAddVeterinaryScreen';
import ComNotificationsScreen from './src/screens/commerces/ComNotificationsScreen';
import ComProductDetailsScreen from './src/screens/commerces/ComProductDetailsScreen';
import ComProductScreen from './src/screens/commerces/ComProductScreen';
import ComServiceDetailsScreen from './src/screens/commerces/ComServiceDetailsScreen';
import ComServiceScreen from './src/screens/commerces/ComServiceScreen';
import ComSettingsScreen from './src/screens/commerces/ComSettingsScreen';
import ComVeterinaryProfileScreen from './src/screens/commerces/ComVeterinaryProfileScreen';
import ComEditProductScreen from './src/screens/commerces/ComEditProductScreen';
import ComEditServiceScreen from './src/screens/commerces/ComEditServiceScreen';

//Imports de Usuario
//pets
import AddPetScreen from './src/screens/users/pets/AddPetScreen';
import PetsScreen from './src/screens/users/pets/PetsScreen';
//products
import ProductScreen from './src/screens/users/products/ProductScreen';
import ProductDetailsScreen from './src/screens/users/products/ProductDetailsScreen';
//profile
import UserProfileScreen from './src/screens/users/profile/UserProfileScreen';
import UserSettingsScreen from './src/screens/users/profile/UserSettingsScreen';
import HistoryScreen from './src/screens/users/profile/HistoryScreen';
import ShopingCartScreen from './src/screens/users/profile/ShopingCartScreen';
//services
import AquariumCleanerScreen from './src/screens/users/services/AquariumCleanerScreen';
import GroomingScreen from './src/screens/users/services/GroomingScreen';
import MassagesScreen from './src/screens/users/services/MassagesScreen';
import PetSittingScreen from './src/screens/users/services/PetSittingScreen';
import PetWalkerScreen from './src/screens/users/services/PetWalkerScreen';
import ServiceDetailsScreen from './src/screens/users/services/ServiceDetailsScreen';
import ServicesScreen from './src/screens/users/services/ServicesScreen';
//veterinaries
import VeterinariesScreen from './src/screens/users/veterinaries/VeterinariesScreen';
import VeterinaryProfileScreen from './src/screens/users/veterinaries/VeterinaryProfileScreen';

import LoadingScreen from './src/screens/authScreens/LoadingScreen';
import SigninScreen from './src/screens/authScreens/SignInScreen';
import SignupScreen from './src/screens/authScreens/SignUpScreen';
test('HomeScreen se renderiza correctamente', () => {
    const tree = HomePageScreen;
    expect(tree).toMatchSnapshot();
});
test('ComHomeScreen se renderiza correctamente', () => {
    const tree = ComHomeScreen;
    expect(tree).toMatchSnapshot();
});
test('VeterinariesScreen se renderiza correctamente', () => {
    const tree = VeterinariesScreen;
    expect(tree).toMatchSnapshot();
});
test('SigninScreen se renderiza correctamente', () => {
    const tree = SignInScreen;
    expect(tree).toMatchSnapshot();
});

test('SignupScreen se renderiza correctamente', () => {
    const tree = SignUpScreen;
    expect(tree).toMatchSnapshot();
});

test('PetScreen se renderiza correctamente', () => {
    const tree = PetsScreen;
    expect(tree).toMatchSnapshot();
});
test('StartScreen se renderiza correctamente', () => {
    const tree = StartScreen;
    expect(tree).toMatchSnapshot();
});
test('ReviewScreen se renderiza correctamente', () => {
    const tree = ReviewScreen;
    expect(tree).toMatchSnapshot();
});
test('ChatScreen se renderiza correctamente', () => {
    const tree = ChatScreen;
    expect(tree).toMatchSnapshot();
});
test('JoinScreen se renderiza correctamente', () => {
    const tree = JoinScreen;
    expect(tree).toMatchSnapshot();
});
test('FriendListScreen se renderiza correctamente', () => {
    const tree = FriendListScreen;
    expect(tree).toMatchSnapshot();
});
test('PostScreen se renderiza correctamente', () => {
    const tree = PostScreen;
    expect(tree).toMatchSnapshot();
});
test('PostDetailScreen se renderiza correctamente', () => {
    const tree = PostDetailScreen;
    expect(tree).toMatchSnapshot();
});
test('AddPostScreen se renderiza correctamente', () => {
    const tree = AddPostScreen;
    expect(tree).toMatchSnapshot();
});
test('myPostScreen se renderiza correctamente', () => {
    const tree = myPostsScreen;
    expect(tree).toMatchSnapshot();
});
test('pinnedPostsScreen se renderiza correctamente', () => {
    const tree = pinnedPostsScreen;
    expect(tree).toMatchSnapshot();
});
test('recommendedPostsScreen se renderiza correctamente', () => {
    const tree = recommendedPostsScreen;
    expect(tree).toMatchSnapshot();
});
test('CommentsScreen se renderiza correctamente', () => {
    const tree = CommentsScreen;
    expect(tree).toMatchSnapshot();
});
test('ComAddProductScreen se renderiza correctamente', () => {
    const tree = ComAddProductScreen;
    expect(tree).toMatchSnapshot();
});
test('ComAddServiceScreen se renderiza correctamente', () => {
    const tree = ComAddServiceScreen;
    expect(tree).toMatchSnapshot();
});
test('ComVeterinaryScreen se renderiza correctamente', () => {
    const tree = ComVeterinaryScreen;
    expect(tree).toMatchSnapshot();
});
test('ComAddVeterinaryScreen se renderiza correctamente', () => {
    const tree = ComAddVeterinaryScreen;
    expect(tree).toMatchSnapshot();
});
test('ComNotificationsScreen se renderiza correctamente', () => {
    const tree = ComNotificationsScreen;
    expect(tree).toMatchSnapshot();
});
test('ComProductScreen se renderiza correctamente', () => {
    const tree = ComProductScreen;
    expect(tree).toMatchSnapshot();
});
test('ComServiceDetailsScreen se renderiza correctamente', () => {
    const tree = ComServiceDetailsScreen;
    expect(tree).toMatchSnapshot();
});
test('ComServiceScreen se renderiza correctamente', () => {
    const tree = ComServiceScreen;
    expect(tree).toMatchSnapshot();
});
test('ComSettingsScreen se renderiza correctamente', () => {
    const tree = ComSettingsScreen;
    expect(tree).toMatchSnapshot();
});
test('ComVeterinaryProfileScreen se renderiza correctamente', () => {
    const tree = ComVeterinaryProfileScreen;
    expect(tree).toMatchSnapshot();
});
test('ComEditProductScreen se renderiza correctamente', () => {
    const tree = ComEditProductScreen;
    expect(tree).toMatchSnapshot();
});
test('ComEditServiceScreen se renderiza correctamente', () => {
    const tree = ComEditServiceScreen;
    expect(tree).toMatchSnapshot();
});
test('AddPetScreen se renderiza correctamente', () => {
    const tree = AddPetScreen;
    expect(tree).toMatchSnapshot();
});
test('ProductScreen se renderiza correctamente', () => {
    const tree = ProductScreen;
    expect(tree).toMatchSnapshot();
});
test('ProductDetailsScreen se renderiza correctamente', () => {
    const tree = ProductDetailsScreen;
    expect(tree).toMatchSnapshot();
});
test('UserProfileScreen se renderiza correctamente', () => {
    const tree = UserProfileScreen;
    expect(tree).toMatchSnapshot();
});
test('UserSettingsScreen se renderiza correctamente', () => {
    const tree = UserSettingsScreen;
    expect(tree).toMatchSnapshot();
});
test('HistoryScreen se renderiza correctamente', () => {
    const tree = HistoryScreen;
    expect(tree).toMatchSnapshot();
});
test('ShopingCartScreen se renderiza correctamente', () => {
    const tree = ShopingCartScreen;
    expect(tree).toMatchSnapshot();
});
test('AquariumCleanerScreen se renderiza correctamente', () => {
    const tree = AquariumCleanerScreen;
    expect(tree).toMatchSnapshot();
});
test('GroomingScreen se renderiza correctamente', () => {
    const tree = GroomingScreen;
    expect(tree).toMatchSnapshot();
});
test('MassagesScreen se renderiza correctamente', () => {
    const tree = MassagesScreen;
    expect(tree).toMatchSnapshot();
});
test('PetSittingScreen se renderiza correctamente', () => {
    const tree = PetSittingScreen;
    expect(tree).toMatchSnapshot();
});
test('PetWalkerScreen se renderiza correctamente', () => {
    const tree = PetWalkerScreen;
    expect(tree).toMatchSnapshot();
});
test('ServiceDetailsScreen se renderiza correctamente', () => {
    const tree = ServiceDetailsScreen;
    expect(tree).toMatchSnapshot();
});
test('ServicesScreen se renderiza correctamente', () => {
    const tree = ServicesScreen;
    expect(tree).toMatchSnapshot();
});
test('VeterinaryProfileScreen se renderiza correctamente', () => {
    const tree = VeterinaryProfileScreen;
    expect(tree).toMatchSnapshot();
});
test('LoadingScreen se renderiza correctamente', () => {
    const tree = LoadingScreen;
    expect(tree).toMatchSnapshot();
});
test('SigninScreen se renderiza correctamente', () => {
    const tree = SigninScreen;
    expect(tree).toMatchSnapshot();
});
test('SignupScreen se renderiza correctamente', () => {
    const tree = SignupScreen;
    expect(tree).toMatchSnapshot();
});
