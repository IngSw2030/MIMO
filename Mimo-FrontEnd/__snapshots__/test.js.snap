// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddPetScreen se renderiza correctamente 1`] = `[Function]`;

exports[`AddPostScreen se renderiza correctamente 1`] = `[Function]`;

exports[`AquariumCleanerScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ChatScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComAddProductScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComAddServiceScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComAddVeterinaryScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComEditProductScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComEditServiceScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComHomeScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComNotificationsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComProductScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComServiceDetailsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComServiceScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComSettingsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComVeterinaryProfileScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ComVeterinaryScreen se renderiza correctamente 1`] = `[Function]`;

exports[`CommentsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`FriendListScreen se renderiza correctamente 1`] = `[Function]`;

exports[`GroomingScreen se renderiza correctamente 1`] = `[Function]`;

exports[`HistoryScreen se renderiza correctamente 1`] = `[Function]`;

exports[`HomeScreen se renderiza correctamente 1`] = `[Function]`;

exports[`JoinScreen se renderiza correctamente 1`] = `[Function]`;

exports[`LoadingScreen se renderiza correctamente 1`] = `[Function]`;

exports[`MassagesScreen se renderiza correctamente 1`] = `[Function]`;

exports[`PetScreen se renderiza correctamente 1`] = `[Function]`;

exports[`PetSittingScreen se renderiza correctamente 1`] = `[Function]`;

exports[`PetWalkerScreen se renderiza correctamente 1`] = `[Function]`;

exports[`PostDetailScreen se renderiza correctamente 1`] = `[Function]`;

exports[`PostScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ProductDetailsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ProductScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ReviewScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ServiceDetailsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ServicesScreen se renderiza correctamente 1`] = `[Function]`;

exports[`ShopingCartScreen se renderiza correctamente 1`] = `[Function]`;

exports[`SigninScreen se renderiza correctamente 1`] = `[Function]`;

exports[`SigninScreen se renderiza correctamente 2`] = `[Function]`;

exports[`SignupScreen se renderiza correctamente 1`] = `[Function]`;

exports[`SignupScreen se renderiza correctamente 2`] = `[Function]`;

exports[`StartScreen se renderiza correctamente 1`] = `[Function]`;

exports[`UserProfileScreen se renderiza correctamente 1`] = `[Function]`;

exports[`UserSettingsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`VeterinariesScreen se renderiza correctamente 1`] = `[Function]`;

exports[`VeterinaryProfileScreen se renderiza correctamente 1`] = `[Function]`;

exports[`myPostScreen se renderiza correctamente 1`] = `[Function]`;

exports[`pinnedPostsScreen se renderiza correctamente 1`] = `[Function]`;

exports[`recommendedPostsScreen se renderiza correctamente 1`] = `[Function]`;
